src/entities/collect.go

```diff
@@ -173,6 +173,10 @@ func collectTop(c Collector) error {
 
 	for key, collectionStats := range topMetrics.Totals {
 		splitKey := strings.SplitN(key, ".", 2)
+		if len(splitKey) != 2 {
```

Comment:
Is there a test for this?

Reply:
I edited the mock response so that it covers the thing changed here. However, creating an explicit test for it would be rather painful given how the tests were written (the collection tests all use a single mock instance that returns hard-coded values). The existing test should catch a regression now though. Do you think that's enough?